src/main/java/uk/gov/hmcts/reform/et/syaapi/models/RespondToApplicationRequest.java

```diff
@@ -22,4 +22,6 @@ public class RespondToApplicationRequest {
     private UploadedDocumentType supportingMaterialFile;
     @JsonProperty("response")
     private TseRespondType response;
+    @JsonProperty("isRespondingToRequestOrOrder")
+    private Boolean isRespondingToRequestOrOrder;
```

Comment:
Don't need to pass from the frontend, we can just check ourselves whether YES.equals(application.getClaimantResponseRequired()) in the code itself.